src/components/PostList/PostList.jsx

```diff
@@ -1,3 +1,9 @@
-export const PostList = () => (
-  <>Put the list here</>
+import { PostInfo } from '../PostInfo';
+
+export const PostList = ({ posts }) => (
+  <div className="PostList">
+    {posts.map(post => (
+      <PostInfo key={post.id} {...post} />
```

Comment:
I recommend you to pass post as one instance 
```suggestion
      <PostInfo key={post.id} post={post} />
```